src/index.html

```diff
@@ -9,6 +9,44 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <div class="moyo-container">
+      <header class="main-header">
+        <img src="images/logo.png" alt="moyo-logo1"
+         class="moyo-logo1">
+        <nav class="main-nav">
+          <ul class="main-nav-list">
+            <li>
+              <a href="#" class="is-active">Apple</a>
+            </li>
+            <li>
+              <a href="#">Samsung</a>
+            </li>
+            <li>
+              <a href="#">Smartphones</a>
+            </li>
+            <li>
+              <a href="#" data-qa="hover" >Laptops &amp Computers</a>
+            </li>
+            <li>
+              <a href="#">Gadgets</a>
+            </li>
+            <li>
+              <a href="#">Tablets</a>
+            </li>
+            <li>
+              <a href="#">Photo</a>
+            </li>
+            <li>
+              <a href="#">Video</a>
+            </li>
+
```

Comment:
redundant empty line between parent and sibling tags (fix everywhere)

Reply:
Not fixed